rocAL/source/pipeline/master_graph.cpp

```diff
@@ -970,7 +970,7 @@ void MasterGraph::output_routine() {
             _sequence_start_framenum_vec.insert(_sequence_start_framenum_vec.begin(), _loader_module->get_sequence_start_frame_number());
             _sequence_frame_timestamps_vec.insert(_sequence_frame_timestamps_vec.begin(), _loader_module->get_sequence_frame_timestamps());
 #endif
-            _ring_buffer.set_meta_data(full_batch_image_names, output_meta_data);
+            _ring_buffer.set_meta_data(full_batch_sample_names, output_meta_data);
             _ring_buffer.push();  // Image data and metadata is now stored in output the ring_buffer, increases it's level by 1
```

Comment:
@swetha097 comment still refers to "image"

Reply:
Changed